### ISSUE TITLE
Note about LDC

### ISSUE THREAD

**Issue by 9il** [libmir/dcv#56]
1. That it is required.
2. `ldmd2` should be used with DUB, not `ldc2`

**Comment by ljubobratovicrelja** [comment]
Yes, thank you. I was gonna make a whole FAQ page on the site to refer to such matters. That may take some time, so until that I should probably add it to the README.
